src/main/java/com/conveyal/datatools/manager/controllers/DumpController.java

```diff
@@ -207,12 +208,13 @@ public static boolean updateSnapshotMetadata (String jsonString) {
 
     /**
      * Load a v2 JSON dump (i.e., objects with the class structure immediately before the MongoDB migration).
+     * @param req
      */
-    private static boolean loadLegacy(String jsonString) {
+    private static boolean loadLegacy(Request req) {
```

Comment:
I think it makes sense here to keep the parameter type a string, but just surround this function with a Spark wrapper method that takes in the request object and handles the halt if an exception is encountered during the JSON load. The reasoning is that I can see this function potentially being called in other contexts (loading a JSON string from a local file rather than a file that comes over the wire).

Reply:
This seems to have been resolved from work in another branch